### DISCUSSION TITLE
[HW6] Can we reuse the implementation we did on previous homework

### DISCUSSION THREAD

**Discussion by JustinKim98** [prosyslab-classroom/cs424-program-reasoning#470] [Q&A] [answered]
### Your Name

김재우

### Question

Can we modify the implementation we've done in previous homework for this hw6? 
(I guess this is the intension of this homework since it says "Extend the bottom-up enumerative synthesizer from the previous homework" but just asking to be sure)

HW6를 위해 이전 과제에서 만들었던 구현을 가져와 변형해 사용해도 되나요? (과제 안내문에 이전이 구현한 bottomp-up 합성기를 확장하라고 되어 있어 그게 과제의 의도라고 생각됩니다만, 혹시 몰라 확인차 여쭤봅니다)

**Comment by spearo2** [comment, marked as answer]
네 재우님,

만약 추가적인 지도사항이 없었다면 재우님의 구현물이 과제 안내문에 적힌 요구사항을 만족하시면 괜찮습니다.

감사합니다.

---

If there were no additional instructions given, it should be fine if your implementation satisfies the instructions in the `README.md`.

Thank you
